source/routing/isochrone.h

```diff
@@ -37,12 +37,12 @@ www.navitia.io
 
 namespace navitia { namespace routing {
 
-struct IsochronException: public recoverable_exception {
-    IsochronException(const std::string& msg): recoverable_exception(msg) {}
-    IsochronException() = default;
-    IsochronException(const IsochronException&) = default;
-    IsochronException& operator=(const IsochronException&) = default;
-    virtual ~IsochronException() noexcept = default;
+struct IsochroneException: public recoverable_exception {
+    IsochroneException(const std::string& msg): recoverable_exception(msg) {}
+    IsochroneException() = default;
+    IsochroneException(const IsochroneException&) = default;
+    IsochroneException& operator=(const IsochroneException&) = default;
+    virtual ~IsochroneException() noexcept = default;
```

Comment:
You should define that method in .cpp
Bit tricky but if you don't do that, the compiler embeds vtable everywhere and compiling is slow enough already.
(I didn't just caught that, my compiler raised a warning)
So in .h:
```
virtual ~IsochronException() noexcept;
```
And in .cpp:
```
IsochronException::~IsochronException() noexcept = default;
```